x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceIndex.java

```diff
@@ -84,6 +84,11 @@ public static XContentBuilder mappings() {
                 .startObject("properties")
                 .endObject()
                 .endObject()
+                .startObject("chunking_settings")
+                .field("dynamic", "false")
```

Comment:
We know every chunk setting has a `strategy` field so add that to the mappings explicitly. 

`dynamic: false` means the document source is stored but not indexed which means it is not searchable. That is what we want to do, but if we define the `strategy` field here we can search by `strategy` as it will be indexed.

Reply:
I can update this to include strategy as well.